TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -2,6 +2,55 @@
 
 The Istio Technical Oversight Committee (TOC) is responsible for cross-cutting product and design decisions.
 
+* [Charter](#charter)
+* [Committee Mechanics](#committee-mechanics)
+* [Committee Meeting](#committee-meeting)
+* [Committee Members](#committee-members)
+
+## Charter
+
+* Technical Project Oversight, Direction & Delivery
+
+    * Set the overall technical direction and roadmap of the project.
+
+    * Resolve technical issues, technical disagreements and escalations within the project.
+
+    * Set the priorities of individual releases to ensure coherency and proper sequencing.
+
+    * Approve declaring a new long-term supported (LTS) Istio release.
+
+    * Approve the creation and dissolution of working groups and approve leadership changes of working groups.
+
+    * Create proposals based on TOC discussions and bring them to the relevant working groups for discussion.
+
+    * Approve the creation/deletion of GitHub repositories, along with other high-level administrative issues around GitHub and our other tools.
+
+* Happy Healthy Community
+
+    * Establish and maintain the overall technical governance guidelines for the project.
+
+    * Decide which sub-projects are part of the Istio project, including accepting new sub-projects and pruning existing sub-projects to maintain community focus
+
+    * Ensure the team adheres to our [code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and respects our [values](https://github.com/istio/community/blob/master/VALUES.md).
+
+    * Foster an environment for a healthy and happy community of developers and contributors.
+
```

Comment:
how about adding

  * The TOC shall not dictate or interfere with the day-to-day work of individual working groups or their decisions.

Reply:
The TOC reserves the right to interfere in the decisions made by working groups, in particular, TOC can override working group decisions if those decisions are wrong. The TOC can also dissolve working groups, kick out leads of working groups, assign new leads, etc. So the TOC actually will dictate and interfere with the day-to-day work of individual working groups. That's its job.

More generally, this document doesn't specify all the things the TOC won't do, just the things it will, we don't want a list of the non-responsibilities of the TOC.

So no, we won't add that.

Reply:
for the record that came from Dan’s example bylaws and may be good to show good will toward the contributors